safe_transaction_service/tokens/tasks.py

```diff
@@ -111,3 +113,33 @@ def fix_pool_tokens_task() -> Optional[int]:
         if number:
             logger.info("%d pool token names were fixed", number)
         return number
+
+
+@app.shared_task()
+@close_gevent_db_connection_decorator
+def update_token_info_from_token_list() -> int:
+    """
+    If there's at least one valid token list with at least 1 token, every token in the DB is marked as `not trusted`
+    and then every token on the is marked as `trusted`
+
+    :return: Number of tokens marked as `trusted`
+    """
+    tokens = []
+    for token_list in TokenList.objects.all():
+        tokens += token_list.get_tokens()
```

Comment:
If one list raises an exception we don't try the other lists right?

I think we should log it (as we are doing) and continue with the other lists.

Reply:
Agree

Reply:
Done